test/backend/input_verification_backend.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+
+##################### input_verification_backend.py ############################
+#
+# Copyright 2022 The IBM Research Authors.
+#
+################################################################################
+
```

Comment:
If onnx can provide a hook for user to provide input for a model in its backend test, it will be easier for us to add the test.

Reply:
Just realized that we can just save the model in backend test and run it with different inputs with our python script without checking the correctness of output.